### PR TITLE
Release/1.2.0

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Description:
1. Added the following PB 
        a. "Extract and Hunt Indicators on SIEM" - Read the csv file
        b. "> Lookup Hash Codes Details" - Looks for hash code details
        c. "> Hunt Indicators on Qradar" - Hunt indicator on Qradar and create events accordingly
2. Added connector which are required by PB.
3. Changes description.

UTC: 
1. Validated that depended connector get installed.
2. Validated get PB "Extract and Hunt Indicators on SIEM" successfully execute over Attachment record "Sample File".
3. Validated that indicator get created and hunted on QRadar.
4. Validated that indicator with hash code get detailed and new indicator get created.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,5 +16,5 @@ Solution Pack examines Advisories sent by various organisations via email with a
 
 # Next Steps
 
-| [Installation](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/setup.md#installation) | [Configuration](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/usage.md) | [Contents](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/contents.md) |
+| [Installation](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/docs/setup.md#installation) | [Configuration](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/docs/usage.md) | [Contents](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/docs/contents.md) |
```
Comment (cs-suyog-jinturkar) [inline README.md]: please give relative path instead of FQDN

Also, please change version to 1.0.0 in readme.md

File: docs/contents.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-| [Home](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/README.md) |
+| [Home](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/README.md) |
```
Comment (cs-suyog-jinturkar) [inline docs/contents.md]: Relative path please

File: docs/usage.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-| [Home](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/README.md) |
+| [Home](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.2.0/README.md) |
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Relative path please

File: README.md (hunk 1 above)
Comment (cs-ankit-gupta) [reply README.md]: Done

File: docs/contents.md (hunk 2 above)
Comment (cs-ankit-gupta) [reply docs/contents.md]: Done

File: docs/usage.md (hunk 3 above)
Comment (cs-ankit-gupta) [reply docs/usage.md]: Done

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**